### ISSUE TITLE
Expose the random ticket number 

### ISSUE THREAD

**Issue by umbernhard** [votingworks/arlo#180]
This way observers can recreate the audit sample, and extract subsamples from the audit. Ticket numbers come from the consistent sampler, and should be published either in the retrieval lists or in round reports. 

This will require changes in `sampler.py`, `app.py` and the data model, and possibly also on the frontend.

**Comment by mcchilders** [comment]
Waiting on fix to show each ballot only once in the retrieval list (see comments on #198)

**Comment by benadida** [comment]
@mcchilders retrieval list should be fixed for duplication within a single round. Note that, right now, if a ballot is re-sampled in round 2, it *will* show up in the retrieval list with only the ticket numbers from round 2. I don't know if that is the behavior you want, or not. 

Pros: you can see that the number of samples is correct in round 2.

Cons: you don't need to re-retrieve the ballot, so maybe there should be a marker that it is already accounted for?

Other than that, please test and report any gaps!

**Comment by mcchilders** [comment]
@ben I think what you did is exactly right - the retrieval lists are round-specific, so they should only show the ticket numbers for that round, but then we need a way to indicate whether the ballot has already been audited. Let's add the "Audited: Y/N" column back to do that.

**Comment by umbernhard** [comment]
Another thing that occurred to me is that merging this PR may break some tests for other sampling methods that we've merged since we opened this PR. Not sure if this PR should fix them or if I should just open another PR on this branch that does that.

**Comment by benadida** [comment]
@umbernhard I'm going to rebase this onto master and we'll make sure all tests pass before we merge.

**Comment by umbernhard** [comment]
sounds good to me, like I said some of the tests will likely fail but should be relatively easy to fix.

**Comment by benadida** [comment]
Alright, this is working with retrieval list column indicating if the ballot has been retrieved already. 

@mcchilders please checkout votingworks/arlo#252 and its deployed server to try it out.

**Comment by benadida** [comment]
Just to be clear, this is the instance to check out: 
https://vx-arlo-staging-pr-252.herokuapp.com/
